tasks/00_ingest/Makefile

```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-all: setup pre task post
+all: clean setup pre task post
```

Comment:
`clean` should be run manually for all tasks, so should probably be removed here. (We should probably add an issue to update each tasks's `setup` script to stop execution if the table it's trying to create already exists and has rows.)

Reply:
yup, good catch. can't remember why i had put the `clean` back in but had removed in a more recent commit that i've yet to push.